README.md

```diff
@@ -3,23 +3,146 @@
 This is the SDK to access Devo directly from Node.js.
 It can be used to query Devo, and to manage deferred tasks.
 
+## Quick Start
+
+Install with `npm`:
+
+    $ npm install @devo/nodejs-sdk
+
+Send some data to Devo in your code:
+
+``` js
+const devo = require('@devo/nodejs-sdk')
+
+const sender = devo.sender({host, port})
+sender.send('my first message')
+```
+
+From that point you can start sending events:
+
+```js
+sender.send('something happened')
+sender.send({message: 'something happened', priority: 'high'})
+```
+
+See detailed info on [sending events](#sending-events),
+[Devo client](#devo-client),
+[querying](#querying) and [task management](#task-management).
+Also check out how to make [command line queries](command-line-queries).
+
 ## Installation
 
-Install with:
+You will need to have Node.js locally installed, version 8 or later.
+Install the SDK with:
 
     $ npm install @devo/nodejs-sdk
 
-## Use
+## Sending Events
+
+The SDK supports sending events to Devo.
+First import the SDK in your code and start the sender:
+
+``` js
+const devo = require('@devo/nodejs-sdk')
+
+const sender = devo.sender(options)
+```
+
+From there you can start sending events:
+
+```js
+sender.send('something happened')
+sender.send({message: 'something happened', priority: 'high'})
+```
+
+Devo only ingests plain text messages;
+if an object is passed it will be converted to JSON before sending it.
+
+### Sender Credentials
+
+You need to be a customer to send your events to Devo.
+You can obtain your certificate and private key from
+[Devo](https://www.devo.com/):
+go to the "Administration/Credentials" section,
+and then to
+["X.509 Certificates"](https://docs.devo.com/confluence/docs/administration/administration-credentials#Administrationcredentials-X.509Certificates).
+
+### Sender Options
+
+These options are passed to the `devo.sender()` constructor.
+They will determine where and how to send the events.
+
+#### `host`
+
+The host to send the events.
+
+* For EU: `app.logtrust.com`.
+* For US: `usa.logtrust.com`.
+
+#### `port`
+
+The port to send the events, mandatory.
+Currently only port 443 is enabled for ingestion.
+
+#### `key`
+
+Private key for sending events securely.
+You need to pass the whole key, not a path.
+
+#### `cert`
+
+Certificate for sending securely.
+You need to pass the whole certificate, not a path.
+
+#### `objectMode`
+
+If the sender is going to be used as a stream and you want to
+send objects, set to `true`. Objects will be converted to JSON.
+
```

Comment:
I am missing an example here. How do you stream objects to the sender? What happens if you stream objects to the sender and the `objectMode` flag is set to `false`?

Reply:
Added example in commit 61d440f.

Reply:
Not necessarily, you can use the `sender` as a stream by using its stream methods, such as [stream.write()](https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback).